messages/create.json

```diff
@@ -27,5 +27,7 @@
   "retryIsNotValidForSandboxes": "One cannot use flag retry with Sandbox org create.",
   "sandboxSuccess": "The sandbox org creation process %s is in progress. Run \"sfdx force:org:status -n %s -u %s\" to check for status. If the org is ready, checking the status also authorizes the org for use with Salesforce CLI.",
   "requiresUsername": "This command requires a username. Specify it with the -u parameter or with the \"sfdx config:set defaultusername=<username>\" command.",
-  "missingLicenseType": "The sandbox license type is required, but you didn't provide a value.  Specify the license type in the sandbox definition file with the \"licenseType\" option, or specify the option as a name-value pair at the command-line.  See https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_sandbox_definition.htm for more information."
+  "missingLicenseType": "The sandbox license type is required, but you didn't provide a value.  Specify the license type in the sandbox definition file with the \"licenseType\" option, or specify the option as a name-value pair at the command-line.  See https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_sandbox_definition.htm for more information.",
+  "dnsTimeout": "The sandbox failed to authenticate in time, if this frequently occurs, trying increasing the SFDX_DNS_TIMEOUT environment variable",
+  "partialSuccess": "The sandbox has been created, try running \"sfdx force:org:list\" to verify, you may have to manually set the alias or config values. Run \"sfdx force:org:status\" to connect"
```

Comment:
The sandbox created and authenticated.  Run `sfdx force:org:list` to verify.  It it isn't there, use `sfdx force:org:status` to connect.

You may have to manually set the alias (`sfdx alias set`) or defaultusername (sfdx config:set)